LICENSE.md

```diff
@@ -1,5 +1,7 @@
 MIT License
 
+## This is a test
```

Comment:
This is a test comment

Reply:
test

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](http://semasoftware.com/gh) &nbsp;**Tags:** Efficient

Reply:
test

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](http://semasoftware.com/gh) &nbsp;**Tags:** Efficient

Reply:
test 777

Reply:
123 number

Reply:
test 8/10/2022

__
[![sema-logo](undefined/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Efficient, Not reusable

Reply:
test 8/18/2022

__
[![sema-logo](undefined/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question&nbsp; | &nbsp;**Tags:** Unreadable, Inelegant, Reusable